using-the-website/staking.md

```diff
@@ -2,9 +2,29 @@
 
 Staking allows you to earn OHM passively via auto-compounding. By staking your OHM with OlympusDAO, you receive sOHM in return at a 1:1 ratio. After that, your sOHM balance will increase automatically on every epoch based on the current APY.
 
+## How to Buy OHM
+
+1. Go to [this Sushiswap swap page](https://app.sushi.com/swap?outputCurrency=0x383518188c0c6d7730d91b2c03a03c837814a899).
+
+2. Make sure the output currency is OHM.
+
+3. You can select any input currency based on your available wallet balance. It is recommended to use DAI as the input currency to minimize the slippage.
+
+<p align="center">
+   <img width="1028" height="477" src="../.gitbook/assets/using-the-website/staking/buy_ohm.png" alt="Make sure the output currency is OHM">
+</p>
+
+4. Select the amount of OHM you want to swap for. Then click "Approve" and sign the transaction.
```

Comment:
I see the link above contains the OHM contract address, but for people not that familiar is there another way of verifying the OHM token you purchase is the right one?

Reply:
i think the safest way is to copy the OHM contract address into the output currency field. Let me see if I can add this to the instructions as well

Reply:
Added extra instruction on pasting the OHM contract address. Let me know what you think

Reply:
Yeah that makes sense. Let's go.